src/main/java/org/aerogear/digger/client/services/BuildService.java

```diff
@@ -144,7 +147,7 @@ public BuildTriggerStatus build(JenkinsServer jenkinsServer, String jobName, lon
             throw new IllegalArgumentException("Unable to find job for name '" + jobName + "'");
         }
 
-        final QueueReference queueReference = job.build();
+        final QueueReference queueReference = params == null ? job.build() : job.build(params);
```

Comment:
I would go with `MapUtils.isEmpty(params)`. Just a better practice to employ in general.